jax/experimental/jax2tf/jax2tf.py

```diff
@@ -41,10 +41,38 @@
 from tensorflow.compiler.tf2xla.python import xla as tfxla  # type: ignore[import]
 from tensorflow.compiler.xla import xla_data_pb2  # type: ignore[import]
 
-# A value suitable in a TF tracing context: tf.Tensor, tf.Var, or
-# Python scalar or numpy.ndarray.
+# A value suitable in a TF tracing context: tf.Tensor, tf.Variable, tf.EagerTensor,
```

Comment:
FYI I don't think `tf.EagerTensor` is a public type in TF.

Reply:
It is not, but it is the type of objects that are passed to primitives when running in eager mode. Hence, I think that the comment is accurate.